### PR TITLE
Doc 598 b 5.7

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by laurynas-biveinis** [comment]
Please close https://github.com/percona/percona-server/pull/2288

File: doc/source/flexibility/binlogging_replication_improvements.rst [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+binlog format, |MySQL| was switching to row-based logging of all statements the
+end of the session or until all temporary tables used in this session are
+dropped. It is inconvenient in case of long lasting connections, including
+replication-related ones. |Percona Server| fixes the situation by switching
+between statement-based and row-based logging as and when necessary.
+
+Version Specific Information
+****************************
+
+  * :rn:`5.6.21-70.1`
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/binlogging_replication_improvements.rst]: 5.7 version info please

File: doc/source/flexibility/binlogging_replication_improvements.rst [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+non-deterministic result or not, and switches from statement-based logging
+to row-based one. |Percona Server| is more accurate, it acknowledges such
+instructions as safe when they include ``ORDER BY PK`` or ``WHERE``
+condition. This fix has been ported from the upstream bug report
+:mysqlbug:`42415` (:psbug:`44`).
+
+Version Specific Information
+****************************
+
+  * :rn:`5.6.13-60.5`
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/binlogging_replication_improvements.rst]: 5.7 version info please

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/flexibility/binlogging_replication_improvements.rst [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+Summary of the fix:
+*******************
+
+|MySQL| considers all ``UPDATE/DELETE/INSERT ... SELECT`` statements with
+``LIMIT`` clause to be unsafe, no matter wether they are really producing
+non-deterministic result or not, and switches from statement-based logging
+to row-based one. |Percona Server| is more accurate, it acknowledges such
+instructions as safe when they include ``ORDER BY PK`` or ``WHERE``
+condition. This fix has been ported from the upstream bug report
+:mysqlbug:`42415`.
```
Comment (fiowro) [inline doc/source/flexibility/binlogging_replication_improvements.rst]: Removed mention of PS-44 issue, because it is internal.

Comment (laurynas-biveinis) [reply doc/source/flexibility/binlogging_replication_improvements.rst]: It was not correct for PS-44 to be internal - I have sanitized it and made public - please restore.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**